src/Microsoft.AspNetCore.Server.Testing/Deployers/ApplicationDeployer.cs

```diff
@@ -141,8 +140,8 @@ private void KillProcess(int processId)
             {
                 startInfo = new ProcessStartInfo
                 {
-                    FileName = "pkill",
-                    Arguments = $"-TERM -P {processId}",
+                    FileName = "kill",
+                    Arguments = $"-TERM {processId}",
                 };
```

Comment:
Strange that this does not work:
![image](https://cloud.githubusercontent.com/assets/2030323/13367073/57781bd0-dc94-11e5-8142-58c69f72b2bc.png)
I will see what else we can do. In the worst case we can combine `pkill -TERM -P {processId}` and `kill -TERM {processId}` to kill the children then kill the parent but I feel that there must be a better way.